<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test</class>
 <widget class="QDialog" name="test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Run">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ResultTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadImage">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>LoadImage</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ImageTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProcNameLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="ImageGraphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>441</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
